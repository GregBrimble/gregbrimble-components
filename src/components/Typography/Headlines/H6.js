import React from 'react';

import styled from 'styled-components';
import {HeadlineSmall} from './shared';

const H6 = props => <HeadlineSmall as={styled.h6`
    font-size: 1.25rem;
    line-height: 2rem;
`} {...props} />;

export default H6;
