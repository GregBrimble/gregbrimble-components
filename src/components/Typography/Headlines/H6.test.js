import React from "react"
import H6 from "./H6"
import renderer from "react-test-renderer"

describe("H6", () => {
  it("renders properly", () => {
    const tree = renderer.create(<H6>Headline 1</H6>).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
