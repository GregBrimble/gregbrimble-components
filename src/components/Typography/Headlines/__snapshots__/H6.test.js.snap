// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`H6 renders properly 1`] = `
<h6
  className="sc-bdVaJa sc-bwzfXH cSXMnb sc-htpNat bCorog"
>
  Headline 1
</h6>
`;
